feature_server/run.py

```diff
@@ -62,17 +70,17 @@ def choose_path(base,top):
     'trusted' : ['trustedpass']
 }
 
-for index, name in enumerate((CONFIG_FILE, 'config.txt.default')):
-    try:
-        config = json.load(open(name, 'rb'))
-        if index != 0:
-            print '(creating config.txt from %s)' % name
-            shutil.copy(name, CONFIG_FILE)
-        break
-    except IOError, e:
-        pass
-else:
-    raise SystemExit('no config file found')
+try:
+    with open(config_file, 'rb') as f:
+        config = json.load(f)
+        cfg.config = config
+except IOError as e:
+    print("Error reading config from {}: ".format(config_file) + str(e))
+    sys.exit(1)
+except ValueError as e:
+    print("Error in config file {}: ".format(config_file) + str(e))
+    sys.exit(1)
+
```

Comment:
config.default.txt is not copied on first launch. but not a big deal, we can merge it with out it